images/virtualization-artifact/pkg/controller/vi/internal/source/errors.go

```diff
@@ -65,18 +65,16 @@ func NewVirtualDiskNotReadyError(name string) error {
 	}
 }
 
-type VirtualDiskAttachedToRunningVMError struct {
-	name   string
-	vmName string
+type VirtualDiskInUseError struct {
+	name string
 }
 
-func (e VirtualDiskAttachedToRunningVMError) Error() string {
-	return fmt.Sprintf("VirtualDisk %q attached to running VirtualMachine %q", e.name, e.vmName)
+func (e VirtualDiskInUseError) Error() string {
+	return fmt.Sprintf("VirtualDisk %q attached to running `VirtualMachine`", e.name)
```

Comment:
Why backticks only on VirtualMachine? Seems unneeded.

Reply:
Also, error is not informative. It should reflect the problem: vi/cvi reconciler can't operate on VD (import from it?) while VD is in use by the VM.
Secondly, "attached to" is replaced by "In use", only this error still has "attached to" =)
Thirdly, VirtualDisk name will be logged as a "name" label, no need to add it to the message.
And finally, can we return VirtualMachine name to indicate which VM is responsible for this error?

So, something like this would be consistent and more informative:
```
fmt.Sprintf("reading from the VirtualDisk is not possible while it is in use by the running VirtualMachine/%s", e.vmName).
```

Reply:
Thanks!

Reply:
I did exactly what you said.